randovania/data/json_data/prime1.txt

```diff
@@ -2164,36 +2320,56 @@ Asset id: 1724639367
 > Dock to Main Ventilation Shaft Section B; Heals? False
   > Main Ventilation Shaft Section B/Dock to Main Ventilation Shaft Section A
       Trivial
+  > Main Ventilation Shaft Section A/Dock to Reactor Core
+      Trivial
 
 > Dock to Reactor Core; Heals? False
   > Reactor Core/Dock to Main Ventilation Shaft Section A
       Trivial
+  > Main Ventilation Shaft Section A/Dock to Main Ventilation Shaft Section B
+      Trivial
 
 ----------------
 Reactor Core
 Asset id: 3993592474
 > Dock to Reactor Access; Heals? False
   > Reactor Access/Dock to Reactor Core
       Trivial
+  > Reactor Core/Dock to Main Ventilation Shaft Section A
+      Space Jump Boots
 
 > Dock to Main Ventilation Shaft Section A; Heals? False
   > Main Ventilation Shaft Section A/Dock to Reactor Core
       Trivial
+  > Reactor Core/Dock to Reactor Access
+      Wave Beam and Thermal Visor
 
 ----------------
 Reactor Access
 Asset id: 4215436672
 > Dock to Cargo Freight Lift to Deck Gamma; Heals? False
   > Cargo Freight Lift to Deck Gamma/Dock to Reactor Access
       Trivial
+  > Reactor Access/Dock to Reactor Core
+      Trivial
+  > Reactor Access/Dock to Savestation
+      Trivial
 
 > Dock to Reactor Core; Heals? False
   > Reactor Core/Dock to Reactor Access
       Trivial
+  > Reactor Access/Dock to Cargo Freight Lift to Deck Gamma
+      Wave Beam and Thermal Visor
+  > Reactor Access/Dock to Savestation
+      Trivial
 
 > Dock to Savestation; Heals? False
   > Savestation/Dock to Reactor Access
       Trivial
+  > Reactor Access/Dock to Cargo Freight Lift to Deck Gamma
+      Wave Beam and Thermal Visor
+  > Reactor Access/Dock to Reactor Core
+      Trivial
 
 ----------------
 Cargo Freight Lift to Deck Gamma
```

Comment:
Missing Pickup node with requires Charge or Missiles. Also, Dock to Reactor Access node requires Space Jump, Wave Beam, Thermal, and Gravity.

Reply:
gravity suit isn't required until cargo freight lift to deck gamma. as for space jump boots it would just be there to prevent having a point of no return

Reply:
I know that. I linked the comment for cargo freight